rdt-app/src/main/java/io/ona/rdt_app/job/RDTJobCreator.java

```diff
@@ -23,8 +25,8 @@
     @Override
     public Job create(@NonNull String tag) {
         switch (tag) {
-            case SyncServiceJob.TAG:
-                return new SyncServiceJob(SyncIntentService.class);
```

Comment:
Its possible to reuse the `SyncServiceJob` and pass a  service which extends  `SyncIntentService.class`
e.g 
 return new SyncServiceJob(RDTSyncIntentService.class);